bats/types.bats

```diff
@@ -0,0 +1,995 @@
+#!/usr/bin/env bats

```

Comment:
Same comment in this very long test file. Anywhere you're only asserting the exit code of a dolt command, just omit "run" and save yourself a line.